setup.py

```diff
@@ -47,7 +47,9 @@
         "click",
         "psutil",
         "gunicorn",
-        "six>=1.14.0"
+        "six>=1.14.0",
+        "bokeh",
+	"selenium"
```

Comment:
Please fix the indentation here. It needs to match the line above it, and should use spaces, not tabs as it does now:
![image](https://user-images.githubusercontent.com/31522931/99725845-4e6c5a80-2a7b-11eb-8085-f3f6643cf6bc.png)

Reply:
@ccarterlandis : I do not get an error when there is data gathered. We do need to improve empty endpoint messaging, but the API is early in development and its more important to get it in front of people than to make it perfect. Fixing the bokeh thing.

Reply:
![image](https://user-images.githubusercontent.com/379847/99837868-d874fb80-2b2d-11eb-8801-285fa945624f.png)